src/Menu.java

```diff
@@ -1,12 +1,30 @@
+import java.io.*;
+import java.util.ArrayList;
 
 public class Menu {
-	public static Item[] itemMenu = new Item[5];
+	public static ArrayList<Item> itemMenu = new ArrayList<Item>();
 	
 	static {
-		itemMenu[0] = new Item(1,"Noodles",15.00);
-		itemMenu[1] = new Item(2,"Soup",7.00);
-		itemMenu[2] = new Item(3,"Maggi",5.00);
-		itemMenu[3] = new Item(4,"Pizza",30.00);
-		itemMenu[4] = new Item(5,"Burger",20.00);
-	};
+		try {
+			FileReader fileReaderObj = new FileReader("./src/main/resources/Items.txt");
+			BufferedReader bufferedReaderObj = new BufferedReader(fileReaderObj);
```

Comment:
### **_Use Try with resource here_** 

```suggestion
		try (FileReader fileReaderObj = new FileReader("./src/main/resources/Items.txt");
				BufferedReader bufferedReaderObj = new BufferedReader(fileReaderObj);) 
```
### 


An even better approach is
```suggestion
		try (BufferedReader bufferedReaderObj = new BufferedReader(new FileReader("./src/main/resources/Items.txt"));) {
```

> Java 7 introduced the try-with-resources statement, which guarantees that the resource in question will be closed. Since the new syntax is closer to bullet-proof, it should be preferred over the older try/catch/finally version.

### Noncompliant Code Example
```
FileReader fr = null;
BufferedReader br = null;
try {
  fr = new FileReader(fileName);
  br = new BufferedReader(fr);
  return br.readLine();
} catch (...) {
} finally {
  if (br != null) {
    try {
      br.close();
    } catch(IOException e){...}
  }
  if (fr != null ) {
    try {
      br.close();
    } catch(IOException e){...}
  }
}
```

### Compliant Solution

```
try (
    FileReader fr = new FileReader(fileName);
    BufferedReader br = new BufferedReader(fr)
  ) {
  return br.readLine();
}
catch (...) {}
```